DESCRIPTION

```diff
@@ -8,6 +8,56 @@ Authors@R: c(
         "Mangiola",
         email = "mangiolastefano@gmail.com",
         role = c("aut", "cre")
+    ),
+    person(
+        "Michael",
+        "Milton",
+        email = "milton.m@wehi.edu.au",
+        role = c("aut", "rev")
+    ),
+    person(
+        "Martin",
+        "Morgan",
+        email = "",
```

Comment:
`email` is only needed for "`cre`" so you can probably omit this.

Reply:
I have added all emails. Should I eliminate all emails except mine. 

Would people prefer to have emails included or excluded?

Reply:
I don't mind having mine. Probably Julie doesn't need to have hers.